modules/@apostrophecms/file-tag/index.js

```diff
@@ -5,6 +5,7 @@ module.exports = {
     pluralLabel: 'apostrophe:fileTags',
     quickCreate: false,
     autopublish: true,
+    versions: true,
```

Comment:
Yeah... we could do this in an improvement in the document-versions module, but this is a lot less typing and harms nothing when it is not present. 👍